client/state/gravatar-status/test/reducer.js

```diff
@@ -48,17 +48,15 @@ describe( 'reducer', () => {
 
 		test( 'returns empty object by default', () => {
 			const state = tempImage( undefined, {} );
-			expect( state ).to.eql( {} );
+			expect( state ).to.eql( null );
```

Comment:
How much work would be to move the assertions from Chai to Jest?

Reply:
Not much actually, done in https://github.com/Automattic/wp-calypso/pull/62066/commits/d0754411c728414a638df3e0772a87942d116f28.